samples/openthread/coprocessor/README.rst

```diff
@@ -72,6 +72,29 @@ You can also use your own application, provided that it supports the Spinel comm
     |thread_hwfc_enabled|
     In addition, the Co-processor sample reconfigures the baud rate to 1000000 bit/s by default.
 
+Diagnostic module
+=================
+
+By default, the Co-processor sample comes with the :kconfig:option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER` :ref:`feature set <thread_ug_feature_sets>` enabled, which allows you to use ``diag`` commands.
+Use these commands to manually check hardware-related functionalities without running a Thread network.
+For example, to ensure radio communication is working when adding a new functionality or during the manufacturing process.
+
+.. note::
+    If you disable the :kconfig:option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER` feature set, you can enable the diagnostic module with the :kconfig:option:`CONFIG_OPENTHREAD_DIAG` Kconfig option.
+
+Additionally for the ``nrf52840dongle_nrf52840`` build target, diagnostic GPIO commands have been enabled by ``openthread_config`` node in :file:`boards/nrf52840dongle_nrf52840.overlay`.
```

Comment:
Might consider reusing the same section from CLI sample.

Reply:
It is actually reused, with minor changes as changing sample name and removing
```
See `Testing diagnostic module`_ section for an example.
````
line as this section isn't included in coprocessor sample.
Or do you think that we could include that here too?

Reply:
Lines 85-97 are exactly the same for both samples. My point is that  we should have a single place for edits and in the other place just instantiate the original one.

Reply:
Ok, added reference to CLI sample with note about difference in command execution